johnwayodi/store-manager
# #161168512 Add New Attendant Page for Admin

#### What does this PR do?
Adds a view where Admin can create a new Store Attendant
#### Description of Task to be completed?
Add functionality to the **_Create Attendant_** button and the sidebar options
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/store-manager.git`
2. Ensure you are in the directory where you cloned this repository, **_store-manager_** by default
3. `git checkout ft-new-attendant-admin-161168512`
4. open attendant-detail-admin.html on browser
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#161168512
#### Screenshot
![new-attendant](https://user-images.githubusercontent.com/10683903/46841242-76cef880-cdcf-11e8-8564-0074848060b4.png)

#### Questions:
Any feedback on what I can improve or change? Drop a comment or suggestion below
